projects/ad-acevsecrdset-sl/Makefile

```diff
@@ -1,7 +1,15 @@
+ifdef REV_A
+CFLAGS += -DREV_A=1
+else
+CFLAGS += -DREV_D=1
```

Comment:
If there will be other Revisions in the future then we might want to keep both flags (REV_A and REV_D). Also drop the explicit initialization with 1 since it is redundant. To keep the approach similar to the rest of the projects, we should do something like this:
```
REV_A = n
REV_D = y
ifeq (y,$(strip $(REV_A)))
CFLAGS += -DREV_A
else
CFLAGS += -DREV_D
endif
```